services/QuillLMS/engines/evidence/lib/evidence/evidence/check.rb

```diff
@@ -78,5 +80,16 @@ def self.checks_to_run(feedback_types)
 
       filtered_checks
     end
+
+    def self.normalize_entry_text(entry)
```

Comment:
Take or leave. You might consider making this is own small `ApplicationService` class ([example)](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/engines/evidence/lib/evidence/evidence/html_tag_remover.rb), since I could see this being reused.

Reply:
Seems reasonable.  I've gone ahea and done this.